docs/querying/sql-functions.md

```diff
@@ -2473,27 +2575,89 @@ Shifts a timestamp forwards or backwards by a given number of time units.
 
 ## TIMESTAMP_TO_MILLIS
 
-`TIMESTAMP_TO_MILLIS(<TIMESTAMP>)`
+Returns the number of milliseconds since epoch for the given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMP_TO_MILLIS(timestamp_expr)`
+* **Function type:** Scalar, date and time
 
-Returns the number of milliseconds since epoch for the given timestamp.
+<details><summary>Example</summary>
+
+The following example converts the `__time` column from the `taxi-trips` datasource into milliseconds since epoch.
+
+```sql
+SELECT 
+  "__time" AS "original_time",
+  TIMESTAMP_TO_MILLIS("__time") AS "miliseconds"
+FROM "taxi-trips"
+LIMIT 1
+```
+
+Returns the following:
+
+| `original_time` | `miliseconds` |
+| -- | -- |
+| `2013-08-01T08:14:37.000Z` | `1375344877000` |
+
+</details>
+
+[Learn more](sql-scalar.md#date-and-time-functions)
 
 ## TIMESTAMPADD
 
-`TIMESTAMPADD(<unit>, <count>, <TIMESTAMP>)`
+Adds a `unit` of time to `timestamp` a total of `count` times.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMPADD(unit, count, timestamp)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example adds five months to the timestamp `2000-01-01 00:00:00`.
```

Comment:
Should the timestamp in the intro sentence and the example that follows be `2000-01-01T00:00:00.000Z` for consistency? To match what is in the results table?

Reply:
No, from what I can tell, when parsing a string as a timestamp it MUST follow a specific format (yyyy-MM-dd) otherwise If I try to use the other method I get the error:

```llegal TIMESTAMP literal '2000-01-01T00:00:00.000Z': not in format 'yyyy-MM-dd HH:mm:ss```